### PR TITLE
Add Cypress custom commands

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING CI && PENDING REVIEW

### What does this PR accomplish?
- Introduces Cypress [custom commands](https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax) to the project
- Creates `cy.icon(icon_name)` command and replaces all occurrences of `cy.get(".Icon-icon_name")`

### How to test this?
- All tests should pass in CI

### Additional notes:
- I can add only one command in this PR if it makes it easier to review since every single command will touch a lot of files?

### CODE REVIEW

File: frontend/test/metabase/scenarios/collections/collections.cy.spec.js [hunk 1]
```diff
@@ -529,6 +529,10 @@ describe("scenarios > collection_defaults", () => {
         .find(".Icon-chevronright")
         .should("not.exist");
     });
+
+    it.only("dasda", () => {
```
Comment (daltojohnso) [inline frontend/test/metabase/scenarios/collections/collections.cy.spec.js:533]: don't forget this guy

**Review by daltojohnso — CHANGES_REQUESTED** [review]

File: frontend/test/metabase/scenarios/collections/collections.cy.spec.js (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/collections/collections.cy.spec.js:533]: Hahahah, great catch! Sorry about that 😊

**Review by daltojohnso — APPROVED** [review]
LGTM

**Review by paulrosenzweig — APPROVED** [review]
Looks great!

> I can add only one command in this PR if it makes it easier to review since every single command will touch a lot of files?

👍 I think splitting commands into separate PRs is a good idea.

**Pull request merged**